I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script2.rpy", line 175, in script
    $ shake()
  File "game/script2.rpy", line 175, in <module>
    $ shake()
NameError: name 'shake' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script2.rpy", line 175, in script
    $ shake()
  File "/Volumes/renpy-8.1.3-sdk/renpy-8.1.3-sdk/renpy/ast.py", line 1138, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "/Volumes/renpy-8.1.3-sdk/renpy-8.1.3-sdk/renpy/python.py", line 1122, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/script2.rpy", line 175, in <module>
    $ shake()
NameError: name 'shake' is not defined

macOS-14.2.1-arm64-arm-64bit arm64
Ren'Py 8.1.3.23091805
Transition of Instances 1.0
Mon Dec 25 19:40:19 2023
